src/libs/antares/study/parameters.cpp

```diff
@@ -1111,15 +1111,30 @@ static bool SGDIntLoadFamily_Legacy(Parameters& d,
 
     if (key == "initial-reservoir-levels") // ignored since 9.2
     {
-        if (version >= StudyVersion(9, 2))
+        if (value == "hot start")
         {
             logs.warning()
               << "Option initial-reservoir-levels is deprecated, please remove it from the study";
         }
-        else if (value == "hot start")
+        return true;
+    }
+
+    if (key == "set-to-null-ntc-between-physical-out-for-first-step") // ignored since 9.2
+    {
+        if (value == "false")
         {
-            logs.warning()
-              << "Hydro hot start not supported with this solver, please use a version < 9.2";
+            logs.warning() << "Parameter set-to-null-ntc-between-physical-out-for-first-step "
+                              " is deprecated, please remove it from the study";
+        }
+        return true;
+    }
+
+    if (key == "enable-first-step") // ignored since 9.2
+    {
+        if (value == "false")
```

Comment:
We can't handle enable-first-step=true anymore, even though it was the default value previously.
```suggestion
        if (value == "true")
```